.travis.yml

```diff
@@ -22,7 +22,7 @@ script:
   - sudo modprobe ip_vs ip_vs_rr
   - docker swarm init --advertise-addr 127.0.0.1
   - docker secret create amplifier_yml tests/test.amplifier.yml
-  - amp cluster create --local-no-metrics
+  - amp cluster create --no-metrics
```

Comment:
btw, it's cool that it's deploying faster for tests, but that means we miss any regression on the metrics layer.
I think we should include it.